packages/remark-stringify/types/test.ts

```diff
@@ -0,0 +1,32 @@
+import remarkStringify = require('remark-stringify')
+import unified = require('unified')
+import {Node, Parent} from 'unist'
+
+const stringifyOptions: Partial<remarkStringify.RemarkStringifyOptions> = {
```

Comment:
with generic support, I don't think this needs to be explicitly cast, `unified().use(remarkStringify, stringifyOptions)` should verify.

Reply:
I'll fix it as soon as unified v8.1.0(with the modified types) is released.

Reply:
released!

Reply:
Updated